resources/etls/LoadApplication.xml

```diff
@@ -98,6 +119,9 @@
             <source schemaName="cds" queryName="ds_bamaantigen" />
             <destination schemaName="cds" queryName="BAMAAntigen" targetOption="truncate" />
         </transform>
+        <transform id="truncate_studyproductmap" type="org.labkey.di.pipeline.TransformTask">
```

Comment:
Why is the truncate task needed here? Shouldn't ClearMappingTask.clear already have taken care of it?

Reply:
I've removed that and added line to ClearMappingTask.